tox.ini

```diff
@@ -1,22 +1,32 @@
 [tox]
-envlist = copying,{py27,py33,py34,py35,py36}-{lint,tests,cli,checksetup}
+envlist = copying,{py27,py33,py34,py35,py36}-{lint,tests,tools,cli,checksetup}
 
 [testenv]
 whitelist_externals =
     cli:   {[testenv:cli]whitelist_externals}
 deps=
     -rtests_utils/test-requirements.txt
     checksetup: {[testenv:checksetup]deps}
+    tools: {[testenv:tools]deps}
 commands=
     tests:      {[testenv:tests]commands}
+    tools:      {[testenv:tools]commands}
     lint:       {[testenv:lint]commands}
     cli:        {[testenv:cli]commands}
     checksetup: {[testenv:checksetup]commands}
     coverage:   {[testenv:coverage]commands}
 
 [testenv:tests]
 commands=
-    py.test --cov=iotlabcli --cov-report=term --cov-report=xml --cov-report=html
+    py.test --cov-append --cov=iotlabcli --cov-report=term --cov-report=xml --cov-report=html
```

Comment:
`--cov-append` can be removed, normally codecov is able to merge the coverage from different testenvs. I fear side effects between different local runs.

Reply:
OK, removed.